pages/index.md

```diff
@@ -44,7 +44,7 @@ We’re looking for candidates passionate about our mission with top-notch softw
 {% for pg in sortedpages %}
 {% if pg.state == 'open' %}
 {% unless pg.path contains 'template'  %}
-* [{{ pg.title }}]({{ site.baseurl }}{{ pg.permalink }})
+* [{{ pg.title }}]({{ site.baseurl }}{{ pg.permalink }}) (Open now through {{ pg.closes | date: '%A, %B %e, %Y at %l:%M%P %Z' }})
```

Comment:
Re-adding the open dates to the homepage.

Reply:
![screen shot 2018-06-05 at 5 10 43 pm](https://user-images.githubusercontent.com/86842/41002986-6915ecae-68e3-11e8-88fd-945dfd458adb.png)